Time - Denisse

```diff
@@ -0,0 +1,83 @@
+# Program that will calculate and display information  
```

Comment:
This code should be in the file worksheet.rb. Most importantly, there should be no spaces in the file title and it needs to have the .rb extension.